ldm/simplet2i.py

```diff
@@ -278,6 +283,18 @@ def process_image(image,seed):
         assert (
             0.0 <= strength <= 1.0
         ), 'can only work with strength in [0.0, 1.0]'
+        assert (
+                0.0 <= variation_amount <= 1.0
+        ), '-v --variation_amount must be in [0.0, 1.0]'
+
+        if len(with_variations) > 0:
+            assert seed is not None,\
```

Comment:
Unfortunately, seed is never None at this point because of line 175. I will see what happens if I take this out.

Reply:
Ok, this can be fixed by moving around where the seed is set. No need to change PR.